ProcessMaker/Http/Controllers/Api/Administration/UsersController.php

```diff
@@ -2,8 +2,13 @@
 
```

Comment:
I'd like to move this controller to User/ProfileController there will be an Admin/UserController for the admin area.